### ISSUE TITLE
MailKit HTMLBody is rendered as plain-text!

### ISSUE THREAD

**Issue by arn3342** [jstedfast/MimeKit#447]
I have a complex HTML email template that consists of many `div`, `section` and other HTML elements. The template also has reference to CSS(uploaded to server). I am using the below code to send the email :

    var message = new MimeMessage();
    var bodyBuilder = new BodyBuilder();
    bodyBuilder.HtmlBody = File.ReadAllText(pathToHtmlFIle);
    message.Body = bodyBuilder.ToMessageBody();

    client.Send(message);

But the client is receiving the email as plain-text only. The email is losing all it's formatting, colors and everything except for the basic texts. Is this the expected result ? How can make MailKit send a full HTML email keeping everything as-it-is?

**Comment by jstedfast** [comment]
You shouldn’t need to do anything else. Can you attach a sample message that was sent by this code? I wonder if maybe the receiving client is just stripping the html?

**Comment by jstedfast** [comment]
I just tested a sample app like yours and the `Content-Type:` header was `text/html`, which was correct and I was able to see html in my mail client.

I was worried that maybe the `BodyBuilder` was setting the `Content-Type:` header to text/plain, but it appears to be working correctly.

Is it possible that the receiving client is stripping the html tags or something?

**Comment by jstedfast** [comment]
Hello? Any update on this?

**Comment by MrChriZ** [comment]
I ran into a problem like this.  Just leaving this as a tip for anyone else that runs into the problem.  
In my case I had encoded the HTML using unicode when storing it in the database.
When I retrieved it I inadvertently retrieved it as UTF-8.
This was obvious when I looked into the HTML with the debugger and saw all the characters had /0 in between them.  These didn't show by the time they reached the mail client.
